cg/cli/analysis.py

```diff
@@ -112,7 +112,7 @@ def link(context, family_id, sample_id):
         # link sample in all its families
         sample_obj = context.obj['db'].sample(sample_id)
         if sample_obj is None:
-            LOG.error(f"Sample '{family}' does not exist.")
+            LOG.error(f"Sample '{sample}' does not exist.")
```

Comment:
1. Should it be sample_id perhaps?
2. Please use %s to outputing when logging to avoid the lint warning
:)

Reply:
In case the linter doesn't say why: https://docs.python.org/3/howto/logging.html#optimization

Reply:
Yes, it should be sample_id.

And about log, I took it from other examples in cg code base to use f-string :-P. I'll change it to %s 😛

Reply:
yeah, this indeed hasn't been applied consistently throughout the code base ;)